### PR TITLE
Added throttle function and test; Added scrollTop "polyfill" function

### PR DESCRIPTION
**Pull request by corescan**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -272,3 +272,72 @@ export function isEqualArray(arr1, arr2, cb) {
...
+    }
+    return result;
+  };
+}
+
+/**
+ * Polyfill for finding document scrollTop.
+ */
+export function getScrollTop() {
+  let scrollTop = pageYOffset;
```
Comment (ThinkTankShark) [inline src/utils.js]: Where do we get the pageYOffset from?

File: src/utils.js [hunk 2]
```diff
@@ -272,3 +272,72 @@ export function isEqualArray(arr1, arr2, cb) {
...
+      timeout = setTimeout(later, remaining);
+    }
+    return result;
+  };
+}
+
+/**
+ * Polyfill for finding document scrollTop.
+ */
+export function getScrollTop() {
```
Comment (ThinkTankShark) [inline src/utils.js]: Could you please add some unit tests to this>

Comment (corescan) [reply src/utils.js]: test added

File: src/utils.js [hunk 3]
```diff
@@ -272,3 +272,72 @@ export function isEqualArray(arr1, arr2, cb) {
...
+      context = args = null;
+    }
+  };
+  return () => {
+    let now = Date.now();
+    if (!previous && options.leading === false) {
+      previous = now;
+    }
+    let remaining = wait - (now - previous);
+    context = this;
```
[the remark this answers is not in the document]
Comment (corescan) [reply src/utils.js]: per conversation earlier: changed arrow functions to normal functions to maintain correct `this`

**Comment by ThinkTankShark** [comment]
@corescan Would you please create a release tag with the notes of what you added after you merged this in master? Appreciate it

**Pull request merged**